Kernel/Process.h

```diff
@@ -319,6 +319,7 @@ class Process
     int sys$realpath(Userspace<const Syscall::SC_realpath_params*>);
     ssize_t sys$getrandom(Userspace<void*>, size_t, unsigned int);
     int sys$setkeymap(Userspace<const Syscall::SC_setkeymap_params*>);
+    int sys$get_keymap_name(Userspace<char*> buffer, size_t buffer_size);
```

Comment:
We don't need a new syscall for this, you can get the current keymap name from `/proc/keymap` :)